bin/template/Dockerfile

```diff
@@ -11,12 +11,12 @@
 FROM node:slim
 
 # Labels for GitHub to read your action
-LABEL "com.github.actions.name"="Your action name"
-LABEL "com.github.actions.description"="A description of your action"
+LABEL "com.github.actions.name"="{{name}}"
```

Comment:
I updated the Dockerfile to be a renderable Mustache template. That way, we can pass this file's contents and a JavaScript object and replace these placeholders with the CLI prompt data.

If there's a lighterweight or simpler choice than using Mustache here, happy to switch it out.

Reply:
I'm a-okay with Mustache - though for this use-case I'd be fine with some simple string-replacements. It'd be far less extensible, but including `mustache` as a dependency for the CLI unfortunately means including it with the library 😞

Could be as simple as:

```dockerfile
LABEL "com.github.actions.name"=":NAME"
```

```js
template
  .replace(':NAME', options.name)
  .replace(':DESCRIPTION', options.description)
```

Reply:
That's a good call, this would definitely the simplest way forward. Will update. 👍